src/lib/challenge.ts

```diff
@@ -19,13 +19,17 @@
  - Name Surname <name.surname@gatesfoundation.com>
 
  - Abhimanyu Kapur <abhi.kapur09@gmail.com>
+ - Raman Mangla <ramanmangla@google.com>
 
  --------------
  ******/
 
 /* istanbul ignore file */
 
-// Flag to igore BDD testing, which will be dealt with in a later ticket
+/*
+ * Flag to ignore BDD testing, which will be dealt
+ * with in a later ticket
```

Comment:
Please write the ticket number so we can know when to remove this comment

Reply:
@eoln mentioned to me a couple of weeks ago that we only need to reference the ticket number in the PR.